public/app/features/connections/pages/DataSourcesListPage.tsx

```diff
@@ -1,11 +1,12 @@
 import * as React from 'react';
 
 import { Page } from 'app/core/components/Page/Page';
+import { DataSourceAddButton } from 'app/features/datasources/components/DataSourceAddButton';
 import { DataSourcesList } from 'app/features/datasources/components/DataSourcesList';
 
 export function DataSourcesListPage() {
   return (
-    <Page navId={'connections-your-connections-datasources'}>
+    <Page navId={'connections-your-connections-datasources'} actions={DataSourceAddButton()}>
```

Comment:
Can we pass the DatasourceAddButton component as is to the actions prop?

Reply:
We can't, because if we do then it's not gonna be rendered.